utils/micro-controllers.stanza

```diff
@@ -186,6 +127,91 @@ public defn MicroController (properties:Tuple<KeyValue>, exist:Tuple<String>, so
   else :
     MicroController(jsons[0])
 
+;============================================================
+;======================= MCU to JITX ========================
+;============================================================
+
+defmethod to-jitx (mcu: MicroController) -> Instantiable :
+  pcb-component my-microcontroller :
+    manufacturer = manufacturer(mcu)
+    mpn = mpn(mcu)
+
+    to-jitx-pin-properties(pin-properties(mcu))
+    val bundle-table = to-jitx-bundles(bundles(mcu))
+    to-jitx-supports(supports(mcu), bundle-table)
+
+    assign-landpattern $
+      switch(mfg-package(mcu)) :
+        ; "SO8N" : SO8N
+        ; "TSSOP14" : TSSOP14
```

Comment:
Why are these commented? Is it just for testing or do we need to implement these land patterns?

Reply:
The commented ones are land patterns for which we haven't yet coded an internal definition. Yep, to be implemented.
Our queries need a way to prioritize land patterns that are implemented.